modules/lib/src/main/resources/assets/js/app/browse/action/EditContentAction.ts

```diff
@@ -1,15 +1,15 @@
-import {ContentTreeGrid} from '../ContentTreeGrid';
-import {EditContentEvent} from '../../event/EditContentEvent';
-import {ContentSummaryAndCompareStatus} from '../../content/ContentSummaryAndCompareStatus';
-import {i18n} from '@enonic/lib-admin-ui/util/Messages';
+import {DefaultErrorHandler} from '@enonic/lib-admin-ui/DefaultErrorHandler';
 import {showWarning} from '@enonic/lib-admin-ui/notify/MessageBus';
+import {i18n} from '@enonic/lib-admin-ui/util/Messages';
+import {ContentSummaryAndCompareStatus} from '../../content/ContentSummaryAndCompareStatus';
+import {EditContentEvent} from '../../event/EditContentEvent';
+import {ContentTreeGrid} from '../ContentTreeGrid';
+import {ContentsLocalizer} from './ContentsLocalizer';
```

Comment:
Can we rename this to `ContentLocalizer` (single)? We usually don't use plural for content actions.